packages/api/lambdas/message-consumer.js

```diff
@@ -177,23 +213,29 @@ function processRecord(record, fromSNS) {
     // normal SNS notification - not a Kinesis fallback
     eventObject = parsed;
     originalMessageSource = 'sns';
-    ruleParam = record.Sns.TopicArn;
+    ruleParam = {
+      collectionName: get(eventObject, 'collection.name', get(eventObject, 'collection')),
```

Comment:
Also, is this updated berhavior tested? Does it need to be?

Reply:
It's tested in Kinesis and SNS integration tests insofar that passing tests mean it is scheduling for the rules correctly. Do you think I should add unit tests?

Reply:
Hmm, I guess since we have the new tests for `queueMessageForRule`, which the consumer uses, then there would be no additional value in testing the Kinesis/SNS consumers for respecting a collection in a CNM notification? If so, then I think I've talked myself out of the need for tests here